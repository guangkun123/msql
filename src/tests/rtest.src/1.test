#
# Just setup the tables for the test.
#

create table test (
        user    char(10) primary key,
        age     int,
        phone   char(20)
)
\p\g


create table test2 (
        user    char(10) not null,
        fullname    char(30)
) \p\g

