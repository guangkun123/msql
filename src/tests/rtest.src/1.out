
Welcome to the miniSQL monitor.  Type \h for help.


mSQL >     ->     ->     ->     ->     ->     -> 
Query buffer
------------

create table test (
        user    char(10) primary key,
        age     int,
        phone   char(20)
)

[continue]
    -> 
Query OK.




mSQL >     ->     ->     ->     ->     -> 
Query buffer
------------
create table test2 (
        user    char(10) not null,
        fullname    char(30)
) 
[continue]
    -> 
Query OK.




mSQL >     ->     -> 
Bye!

